content/docs/details/accessing-aws-resources/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Accessing AWS Services
```

Comment:
This page doesn't seem to have an entry in the sidebar. You likely need to update `docs-nav.yaml`.

![Screenshot 2022-03-29 at 16 28 02](https://user-images.githubusercontent.com/562403/160648145-1b1a218c-7115-4db6-b406-f2ca0216ff42.png)